examples/use_rust/nrf52840_ble_split/memory.x

```diff
@@ -1,13 +1,11 @@
 MEMORY
 {
   /* NOTE 1 K = 1 KiBi = 1024 bytes */
-  /* FLASH : ORIGIN = 0x00000000, LENGTH = 1024K */
-  /*RAM : ORIGIN = 0x20000000, LENGTH = 256K */
-
-  /* These values correspond to the NRF52840 with Softdevices S140 6.1.1 */
-  /* FLASH : ORIGIN = 0x00026000, LENGTH = 824K */
+  /* These values correspond to the nRF52840 WITH Adafruit nRF52 bootloader */
+  FLASH : ORIGIN = 0x00001000, LENGTH = 1020K
```

Comment:
If I understand this correctly the idea is to override the SoftDevice part of the of the Adafruit bootloader. Laaooking at the adafruit boot loader it seems to require around 40k?

Reply:
yes, you're correct. This value comes from https://devzone.nordicsemi.com/f/nordic-q-a/97314/uf2-image-and-bootloader. I'd check it again, ensuring it's correct

Reply:
Checked again. The bootloader size is 40KB, but it's placed at 0xF4000, according to 

https://github.com/adafruit/Adafruit_nRF52_Bootloader/blob/4329614a8f2a020b31af36a1f3fd843d357eb619/linker/nrf52840.ld#L8-L14

so it's correct.